api/controllers.js

```diff
@@ -6,14 +6,14 @@ module.exports = {
   /// USER CONTROLLERS ///
 
   getProfile: (req, res) => {
-    if (!req.query.userId) {
-      res.status(400).send('Missing param "userId"');
+    if (!req.query.email) {
```

Comment:
This is just switching the primary lookup for profile info to being based on email instead of an ID?

Reply:
Yes